plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/editors/data/preferences/PrefPageResultSetMain.java

```diff
@@ -314,6 +324,7 @@ protected void clearPreferences(DBPPreferenceStore store) {
 
         store.setToDefault(ResultSetPreferences.KEEP_STATEMENT_OPEN);
         store.setToDefault(ResultSetPreferences.RS_EDIT_USE_ALL_COLUMNS);
+        store.setToDefault(ResultSetPreferences.RS_EDIT_KEY_VALIDATION_STRATEGY);
```

Comment:
Looks like we don't have a default value defined anywhere

Reply:
The default value is `null`.